.gitignore

```diff
@@ -39,5 +39,6 @@ frontend/.svelte-kit
 /letsencrypt
 backend/json.gz_dumps
 internal/tmp
+internal/logs.txt
```

Comment:
Does this file get updated outside of the container when working with it locally?

Reply:
yep, it gets updated outside the container

Reply:
Sweet!